### PR TITLE
Add end to end workflow example with feature encoding and ridge regression

### PR DESCRIPTION
**Pull request by RFinkelberg**
Contributes an example workflow on the GDELT dataset showcasing
- ETL using dask_cudf
- Feature engineering and encoding (min/max scaling, one hot encoding, label encoding)
- Ridge regression training (with train_test_split)
- Model evaluation (MSE loss calculation, learning curve visualization)

### CODE REVIEW

**Comment by taureandyernv** [comment]
@RFinkelberg, is this still WIP or is it completed?  If completed, i'll test and merge it, if not, please edit the title to say [WIP].  Thanks!

**Comment by RFinkelberg** [comment]
Should be completed and good to merge. Let me know if you run into any problems
